datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1318,4 +1327,405 @@ enum ChartQueryType {
+
+    """
+    Caveat Description
+    """
+    caveatDescription: String
+
+    """
+    Relevant groups that were not represented in the evaluation dataset?
+    """
+    groupsNotRepresented: [String]
+}
+
+type EthicalConsiderations {
+
+    """
+    Does the model use any sensitive data (e.g., protected classes)?
+    """
+    data: [String]
+
+    """
+    Is the model intended to inform decisions about matters central to human life or flourishing – e.g., health or safety? Or could it be used in such a way?
+    """
+    humanLife: [String]
+
+    """
+    What risk mitigation strategies were used during model development?
+    """
+    mitigations: [String]
+
+    """
+    What risks may be present in model usage? Try to identify the potential recipients, likelihood, and magnitude of harms. If these cannot be determined, note that they were considered but remain unknown.
+    """
+    risksAndHarms: [String]
+
+    """
+    Are there any known model use cases that are especially fraught? This may connect directly to the intended use section
+    """
+    useCases: [String]
+}
+
+type BaseData {
+
+    """
+    Dataset used for the Training/Evaluation of the MLModel
+    """
+    dataset: String
+
+    """
+    Motivation to pick these datasets
+    """
+    motivation: String
+
+    """
+    Details of Data Proprocessing
+    """
+    preProcessing: [String]
+}
+
+type Metrics {
+
+    """
+    Measures of MLModel performance
+    """
+    performanceMeasures: [String]
+
+    """
+    Decision Thresholds used (if any)?
+    """
+    decisionThreshold: [String]
+
+}
+
+type IntendedUse {
+
+    """
+    Primary Use cases for the model.
+    """
+    primaryUses: [String]
+
+    """
+    Primary Intended Users
+    """
+    primaryUsers: [IntendedUserType]
+
+    """
+    Out of scope uses of the MLModel
+    """
+    outOfScopeUses: [String]
+}
+
+enum IntendedUserType {
+
+    ENTERPRISE
+
+    HOBBY
+
+    ENTERTAINMENT
+}
+
+type SourceCode {
+    """
+    Source Code along with types
+    """
+    sourceCode: [SourceCodeUrl]
+}
+
+type SourceCodeUrl {
+
+    """
+    Source Code Url Types
+    """
+    type: SourceCodeUrlType
+
+    """
+    Source Code Url
+    """
+    sourceCodeUrl: String
+}
+
+enum SourceCodeUrlType {
+
+    ML_MODEL_SOURCE_CODE
+
+    TRAINING_PIPELINE_SOURCE_CODE
+
+    EVALUATION_PIPELINE_SOURCE_CODE
+}
+
+type Cost {
+
+    """
+    Type of Cost Code
+    """
+    costType: CostType
+
+    """
+    Code to which the Cost of this entity should be attributed to i.e. organizational cost ID
+    """
+    costValue: CostValue
+}
+
+type CostValue {
+
+    """
+    Organizational Cost ID
+    """
+    costId: Float
+
+    """
+    Organizational Cost Code
+    """
+    costCode: String
+}
+
+enum CostType {
+
+    """
+    Org Cost Type to which the Cost of this entity should be attributed to
+    """
+    ORG_COST_TYPE
+}
+
+type Deprecation {
+    """
+    Whether the dataset has been deprecated by owner
```

Comment:
Dataset? We should replace this with "Whether the entity has been deprecated by owner"

Reply:
Same for comments below :)